hack/after-build/update-generated-deep-copies.sh

```diff
@@ -53,27 +56,24 @@ EOF
 // AUTO-GENERATED FUNCTIONS END HERE
 EOF
 
-	gofmt -w -s "$TMPFILE"
+	env GOPATH=$(godep path):$GOPATH goimports -w "$TMPFILE"
 	mv "$TMPFILE" `result_file_name ${version}`
 }
 
 function generate_deep_copies() {
-  local versions="api v1"
-  # To avoid compile errors, remove the currently existing files.
-  for ver in ${versions}; do
-    rm -f `result_file_name ${ver}`
-  done
-  apiVersions=""
-  for ver in ${versions}; do
-    # Ensure that the version being processed is registered by setting
-    # KUBE_API_VERSIONS.
-    if [ "${ver}" != "api" ]; then
-      apiVersions="${ver}"
-    fi
-    KUBE_API_VERSIONS="${apiVersions}" generate_version "${ver}"
-  done
+	local versions="$@"
```

Comment:
This broken the {verify/update}-generated-deep-copies.sh function as versions is empty pretty much always.  I'm assuming you thought about this with DEFAULT_VERSIONS and VERSIONS below, but you didn't plumb those through...